README.md

```diff
@@ -92,7 +92,7 @@ Decoding of bulk data will be attempted using the specified encoding and
 error handler. If the error handler is `'strict'` (the default), a
 `UnicodeDecodeError` is raised when data cannot be decoded. This is identical
 to Python's default behavior. Other valid values to `errors` include
-`'replace'`, `'ignore'`, and `'backslashreplace'`. See [Unicode HOWTO: The String Type](https://docs.python.org/3/howto/unicode.html#the-string-type) for more information on the behavior of these error handlers.
+`'replace'`, `'ignore'`, and `'backslashreplace'`. See [Unicode HOW-TO: The String Type](https://docs.python.org/3/howto/unicode.html#the-string-type) for more information on the behavior of these error handlers.
```

Comment:
Personally, I'd suggest adding an exception for the reasons I note in https://github.com/valkey-io/libvalkey-py/pull/30#issuecomment-3420050126 as opposed to changing the spelling against the wishes of the linked document.

That said, it's just my personally opinion.